src/operations/SymbolicOperation.cpp

```diff
@@ -0,0 +1,286 @@
+#include "operations/SymbolicOperation.hpp"
+
+#include "Definitions.hpp"
+#include "operations/StandardOperation.hpp"
+
+#include <variant>
+
+namespace qc {
+
+    void SymbolicOperation::storeSymbolOrNumber(const SymbolOrNumber& param, std::size_t i) {
```

Comment:
```suggestion
    void SymbolicOperation::storeSymbolOrNumber(const SymbolOrNumber& param, const std::size_t i) {
```

Reply:
Reminds me of a famous Haiku:

The steam is running
The autumn leaves are falling
The const doesn't care

Reply:
That would be perfect for a Twitter post ;o)